MastodonSDK/Sources/MastodonCore/Service/API/APIService+Account.swift

```diff
@@ -199,3 +200,43 @@ extension APIService {
         return response
     }   // end func
 }
+
+extension APIService {
+    public func fetchUser(username: String, domain: String, authenticationBox: MastodonAuthenticationBox)
```

Comment:
This could be cool to use for handling taps on mentions in profile bios too! (definitely a separate PR tho)